core/src/main/java/com/digitalpebble/storm/crawler/bolt/ParserBolt.java

```diff
@@ -269,8 +310,8 @@ public void execute(Tuple tuple) {
                 urlOL = tmpURL.toExternalForm();
             } catch (MalformedURLException e) {
                 LOG.debug("MalformedURLException on {}", l.getUri());
-                eventCounter.scope(
-                        "error_out_link_parsing_"
+                eventCounter
+                        .scope("error_outlink_parsing_"
```

Comment:
I changed the name of the metric here to have a consistent spelling for "outlink"